docs/src/examples/components/Reaction/BestPractices/ReactionBestPractices.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+
+const doList = [
+  {
+    key: 'one',
+    content:
+      'Use actionable components (for example Button) if the reactions need to be actionable.',
+  },
+  {
+    key: 'two',
+    content:
+      'Add textual representation to the icon slot if it only contains an icon (using title, aria-label or aria-labelledby props on the slot).',
```

Comment:
For accessibility best practices, when would I use title vs aria-label vs aria-labelledby? Or is there an accessibility FAQ page that would be helpful for answering questions like this and just linking to?

Reply:
These are more basic details about accessibility.
- `title` you use when you want the description of the element to be shown in a tooltip when you hover the element  as well as to be read by the screen reader
- `aria-label` is used for screen reader (no tooltip is shown by default out of it)
- `aria-labelledby` is used when the element refers to some other element
I found this nice description here:
https://stackoverflow.com/questions/22039910/what-is-aria-label-and-how-should-i-use-it